com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs

```diff
@@ -4115,6 +4298,12 @@ void RenderDebugViewMaterial(CullingResults cull, HDCamera hdCamera, ScriptableR
             }
         }
 
+        private RTHandle GetDebugViewTargetBuffer()
+        {
+            // Used to contain a test whether to return a float16 target, but no longer needed
+            return m_CameraColorBuffer;
```

Comment:
Yes, this function should be removed in particular as it will ease the forward port on master